src/components/organisms/pupil/OakPupilJourneyUnitsFilter/OakPupilJourneyUnitsFilter.test.tsx

```diff
@@ -9,31 +9,30 @@ import renderWithTheme from "@/test-helpers/renderWithTheme";
 import { oakDefaultTheme } from "@/styles";
 
 const menuItems = [
-  { text: "All", id: 0 },
-  { text: "Biology", id: 1 },
-  { text: "Chemistry", id: 2 },
-  { text: "Physics", id: 3 },
+  { displayText: "All", value: "all" },
```

Comment:
Changing this to a non-indexed approach which is consistent with the approach for underlying RadioGroup component